{
  try {
    ctx.close();
  }
 catch (  NamingException e) {
    LOG.warn("Exception when closing LDAP context:",e);
  }
}
